lib/github_metrics/pr_age_reporter.rb

```diff
@@ -0,0 +1,58 @@
+# rubocop:disable Rails/Date, Rails/TimeZone, Rails/Output
+require 'octokit'
+require 'date'
+
+module GithubMetrics
+  class PrAgeReporter
+    attr_reader :tot
+
+    # rubocop:disable Metrics/AbcSize
+    def call
+      numerator = 0
+      sprint_pull_requests.each do |pr|
+        pr_open_seconds = (pr.closed_at || pr.merged_at || Time.now).to_i - pr.created_at.to_i
+        numerator += pr_open_seconds
+        puts "'#{pr.title}' open for #{seconds_to_hours(pr_open_seconds)} hours"
```

Comment:
Would ActiveSupport::Duration work here?

`[5] pry(main)> d = ActiveSupport::Duration.build(3600)
=> 1 hour`

Reply:
It could, but I intentionally avoided including rails or ActiveSupport here because they are pretty heavy